vpr/src/place/noc_place_utils.cpp

```diff
@@ -478,7 +735,7 @@ bool check_for_router_swap(int user_supplied_noc_router_swap_percentage) {
      * we now only swap router blocks for the percentage of time the user
      * supplied.
      * */
-    return (vtr::irand(99) < user_supplied_noc_router_swap_percentage) ? true : false;
+    return (vtr::irand(99) < user_supplied_noc_router_swap_percentage);
```

Comment:
Could delete if you're pretty sure we won't use this in the future ... could also leave for now.